### DISCUSSION TITLE
The most correct way to get a simple string?

### DISCUSSION THREAD

**Discussion by vresetnikov** [scurker/currency.js#405] [Q&A] [answered]
Hello, silly question perhaps, but what would be the most correct way to get the value of type string from the currency object? Just a raw string. For example: 3.32 => "3.32" . Without a symbol or anything else? Thank you

**Comment by scurker** [comment, marked as answer]
Depending on how you're using it, you may not have to do anything:

```js
const value = currency(1234.56)
// The value below will automatically be coerced to the string value of "1234.56"
document.getElementById('amount').value = value
```

Otherwise, you can call `.toString()` directly, or `.format()` if you want to have a formatted string value with a currency symbol and group separators.
